app/models/forem_instance.rb

```diff
@@ -0,0 +1,9 @@
+class ForemInstance
+  def self.deployed_at
+    ApplicationConfig["RELEASE_FOOTPRINT"].presence || ENV["HEROKU_RELEASE_CREATED_AT"].presence
```

Comment:
For the Forem.cloud environment RELEASE_FOOTPRINT will be set whereas for a Heroku environment HEROKU_RELEASE_CREATED_AT will be set

Reply:
Since both are taken from static values that can never change during the lifetime of the app (you literally need to restart the app to change environment variables), I think the result of the `OR` can be frozen into a constant. We don't need to call a method each time:

`ForemInstance::DEPLOYED_AT` <- this value should never change during the lifetime of the deployed version

Reply:
✅

Reply:
Tbh, I still prefer the method, we could always cache it. Frequently reaching into constants defined by other models seems a bit off to me.

```ruby
def self.deployed_at
  @deployed_at ||=  ApplicationConfig["RELEASE_FOOTPRINT"].presence || ENV["HEROKU_RELEASE_CREATED_AT"].presence
end
```

Reply:
I spent time considering the pro's and cons of each approach, and I don't think theres a right or wrong answer here (especially using the cached method strategy), but it's rather a matter of preference. 

I had put this PR on hold yesterday evening after struggling with testing the variables. After discussion with Rhymes, we had concluded that it was testing essentially an "or" statement and so it would be relatively safe to remove the tests if we weren't able to stub it out.

With a fresh mind this morning I tackled it again and after reading the comment above I've decided to move it back to a method with caching. This will help us to keep these tests in place as well.